src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -55,19 +54,12 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
    * Schedule work according to the schedule of the `backoff`.
    */
   private def scheduleWork(work: => Unit): Unit = {
-    val interval = backoff.nextBackOffMillis().millis
-    context.system.scheduler.scheduleOnce(interval) {
+    val interval = backendCall.pollBackoff.next.backoffMillis
+    context.system.scheduler.scheduleOnce(interval.millis) {
       work
     }
```

Comment:
Is this right?  In the previous version of the code `backoff` encapsulated mutable state, but now there's the concept of `next` in the ADT which returns a potentially different `Backoff` implementation.  It looks like this code will always use the same initial backoff.